docs/source/gbd2017_models/causes/neoplasms/lung_cancer/cancer_model.rst

```diff
@@ -349,68 +349,98 @@ State and Transition Data Tables
      - TBL cancer sequelae prevalence from GBD 2019
      - Not forecasted
    * - MST
-     - 2.06 years (95% CI: 0.42 - 3.83); normal distrbution of uncertainty at draw level
+     - Uniform distribution between 2.06 and 5.38 years
```

Comment:
We're going from a normal distribution with mean 2.06 to a uniform distribution with the minimum at 2.06?